prdoc/pr_6027.prdoc

```diff
@@ -0,0 +1,9 @@
+title: Remove pallet::getter from pallet-offences
+doc:
+  - audience: Runtime Dev
+    description: |
+      This PR removes pallet::getter from pallet-offences from type Reports. It also adds a test to verify that retrieval of Reports still works with storage::getter.
+
+crates:
+  - name: pallet-offences
+    bump: patch
```

Comment:
Removing the getter is a breaking (major) change unless you add a function to the RuntimeAPI with the same signature. I'd go with adding the new function but marking it as deprecated to avoid the breakage but letting people know that it is deprecated and that they should use `Reports::<T>::get(id)` instead

Reply:
Done. Looks better now or can it be simplified?

Reply:
patch should be good